flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/combined/CombinedNtProperties.java

```diff
@@ -0,0 +1,66 @@
+package com.flash3388.frc.nt.combined;
+
+import edu.wpi.first.networktables.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CombinedNtProperties {
+    private final Map<String, NetworkTableEntry> NtEntryMap;
+    private final NetworkTable table;
+
+    public CombinedNtProperties(String name) {
+        this(NetworkTableInstance.getDefault(), name);
+    }
+
+    public CombinedNtProperties(NetworkTableInstance tableInstance, String name) {
+        NtEntryMap = new HashMap<>();
+        table = tableInstance.getTable(name);
+    }
+
+    public void addBooleanEntry(String name, boolean initialValue) {
```

Comment:
It's more `put` than `add`, if we're using `map` terminology

Reply:
Not sure if it should be in the map terminology now...